### DISCUSSION TITLE
Navmesh debug rendering

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#608] [Research]
Suggested by @F1F7Y 

r5reloaded already has some stuff working, maybe we can use their stuff as a basis? https://www.kawe-dev.com/navmesh-renderer

Related PR:
- https://github.com/R2Northstar/NorthstarLauncher/pull/626

**Comment by ASpoonPlaysGames** [comment]
primedev already has this i think?

**Reply by ASpoonPlaysGames** [reply]
https://github.com/F1F7Y/NorthstarPrime/blob/e49d88f785372edf0821ed938b93dd28b6e36a9e/primedev/game/server/ai_helper.cpp looks like it

**Reply by GeckoEidechse** [reply]
Oh sweet. We should definitely carry that over then once we get to it :D

**Comment by GeckoEidechse** [comment]
Completed with https://github.com/R2Northstar/NorthstarLauncher/pull/626
